flocker/docs/version_code_block.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Sphinx extension to add a ``version-code-block`` directive
+
+This directive allows Flocker's release version to be inserted into code
+blocks.
+
+.. version-code-block:: console
+
+   $ brew install flocker-|RELEASE|
+"""
+
+from sphinx.directives.code import CodeBlock
+
+from flocker import __version__ as version
+from flocker.docs import parse_version
+
+
+class VersionCodeBlock(CodeBlock):
+    """
+    Similar to CodeBlock but replaces |RELEASE| with the latest release
+    version.
+    """
+    def run(self):
+        parsed_version = parse_version(version)
+        latest = parsed_version.release
+
+        if parsed_version.weekly_release is not None:
```

Comment:
When I wrote that comment, I was assuming that it was what was wanted. My point was, don't try and generate the same thing as sphinx does, but defer to what sphinx is configured to use (so that there is only one place that needs to be changed, to change what gets displayed).

In any case, the version should be based on the version that sphinx has. And if there has to be logic to format the version, it should live in `flocker.docs._version`.